tests/gocase/unit/type/list/list_test.go

```diff
@@ -93,7 +94,7 @@ func TestZipList(t *testing.T) {
 	defer srv.Close()
 	ctx := context.Background()
 	rdb := srv.NewClientWithOption(&redis.Options{
-		MaxRetries: -1,
+		ReadTimeout: 10 * time.Second,
```

Comment:
@vmihailenco could you help with reviewing whether this option is a good choice?

Reply:
Yes, this seems worth a try. The default is 3 seconds.

In go-redis tests we also have a busy loop to wait until Redis Server is fully bootstrapped, but I am not sure whether it is needed for kvrocks:

```go
for i := 0; i < 100; i++ {
  if err := rdb.Ping().Err(); err != nil {
    time.Sleep(100*time.Millisecond)
    continue
  }
  break
}
```